test/test_restraint_base.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import print_function, division
+import IMP
+import IMP.algebra
+import IMP.core
+import IMP.pmi
+import IMP.pmi.restraints
+import IMP.test
+
+
+class RestraintBaseTestCase(IMP.test.TestCase):
```

Comment:
The general convention in IMP (although we probably don't write it down anywhere) is that our unittest classes are called `Tests` (except in the - very rare - occasions where there are multiple classes in a single test file). This convention is poorly followed in PMI though.